README.md

```diff
@@ -7,7 +7,7 @@ Project Lead: Olivia MacDougal
 Team Members: 
 * Alice Dempsey
 * David Massart
-* Jenna Fleischer
+* Jenna Fleischer - it me
```

Comment:
Since this was just for practice, can you delete this addition? We want to keep master clean, even if it is just the Readme.